scripts/python/database.py

```diff
@@ -58,19 +58,18 @@
     image_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
     name TEXT NOT NULL UNIQUE,
     camera_id INTEGER NOT NULL,
-    prior_qw REAL,
-    prior_qx REAL,
-    prior_qy REAL,
-    prior_qz REAL,
-    prior_tx REAL,
-    prior_ty REAL,
-    prior_tz REAL,
     CONSTRAINT image_id_check CHECK(image_id >= 0 and image_id < {}),
     FOREIGN KEY(camera_id) REFERENCES cameras(camera_id))
 """.format(
     MAX_IMAGE_ID
 )
 
+CREATE_LOCATION_PRIORS_TABLE = """CREATE TABLE IF NOT EXISTS location_priors (
```

Comment:
nit: Shall we call it "PosePrior", so it becomes easier to extend it to also hold a orientation prior in the future? For now, we keep it simple and only store the position.

Reply:
Ok, do you suggest replacing all instances of LocationPrior/location_prior by PosePrior/pose_prior?

Reply:
Done